### PR TITLE
Alen/mc gateway auth example

### PR DESCRIPTION
**Pull request by deusxanima**
created initial bel-examples reference for configuring linkerd gw-mode multicluster with granular auth policies

### CODE REVIEW

**Comment by deusxanima** [comment]
Once you approve, I'll clean up the merge conflicts and merge

File: gitops/flux/helm-example/apps/linkerd-buoyant/helmrelease-linkerd-buoyant.yaml
Comment (southsidedean) [inline gitops/flux/helm-example/apps/linkerd-buoyant/helmrelease-linkerd-buoyant.yaml:1]: Should the `linkerd-buoyant` version be `0.31.0`?  Isn't that the latest?

File: gitops/terraform/environments/example/main.tf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+module "cert_manager" {
+  source    = "../../modules/cert-manager"
+  namespace = local.namespaces.cert-manager
+}
+
+module "linkerd" {
+  source            = "../../modules/linkerd"
+  namespace         = local.namespaces.linkerd
+  chart_version           = "2.15.4"
```
Comment (southsidedean) [inline gitops/terraform/environments/example/main.tf:17]: Should this be 2.15.5?

**Review by southsidedean — CHANGES_REQUESTED** [review]
Looks great, but there are a couple of versions I flagged that might need attention.  Please check those before you merge.

**Review by deusxanima — COMMENTED** [review]
fixed requested changes

**Review by southsidedean — APPROVED** [review]
Looks great!

**Pull request merged**